packages/components/context-menu/sub-components/sub-menu.js

```diff
@@ -70,7 +74,9 @@ const SubMenu = (props) => {
     ) {
       subMenuRef.current.style.left = -1 * subListWidth + "px";
     } else {
-      subMenuRef.current.style.left = itemOuterWidth + "px";
+      theme.interfaceDirection === "rtl"
+        ? (subMenuRef.current.style.right = itemOuterWidth + "px")
+        : (subMenuRef.current.style.left = itemOuterWidth + "px");
```

Comment:
Move the repeating condition (itemOuterWidth + "px") to a separate variable.

Reply:
moved to a separate variable